lib/repl.ts

```diff
@@ -8,19 +8,16 @@
  * @license MIT
  */
 
-'use strict';
-
-const fs = require('fs');
-const path = require('path');
-const net = require('net');
-const repl = require('repl');
+import * as fs from 'fs';
```

Comment:
This generates a Node warning in modern Node.

Reply:
Which version of Node are you using? Should I just be on the latest?

Reply:
Actually, went and prefixed everything. LMK if you have a better way

Reply:
Wow yeah that's the wrong solution.

Reply:
5de019ac3d283e8685d is the right solution.

Reply:
Done!